doc/JSON_INFO.md

```diff
@@ -832,6 +832,7 @@ When you sort your inventory by category, these are the categories that are disp
 | `freeze_point`   | Freezing point of this material (F). Default 32 F ( 0 C ).
 | `edible`   | Optional boolean. Default is false.
 | `rotting`   | Optional boolean. Default is false.
+| `fungalizeable`   | Optional boolean. Default is false.
```

Comment:
```suggestion
| `fungalizeable`   | Optional boolean. Wether or not monster made of this material can be damaged by fungalisation. Default is false.
```

It's a bit weird that sensitivity to fungus depends on a monter property to turn into fungus OR of the material they're made of. Maybe we should drop the material thing and just have a "FUNGAL_DMG_IMMUNE" flag or something like that on monster directly.

Reply:
```suggestion
| `fungalizeable`   | Optional boolean. Whether or not a monster made of this material can be damaged by fungalization. Default is false.
```
Possibly a SPECIES property instead, with a per-monster flag that can override? Admittedly, some critters are down as more than one species, such as both HUMAN and ZOMBIE (which I have always found rather confusing...).

Reply:
I didn't like the material limitation but I did it like that to keep it the same as its original implementation. I prefer the fungal damage immune flag but it would require adding the flag to a lot of other monsters, which I guess doesn't need to be done in this PR.